bootstrap/pkg/kfapp/ksonnet/ksonnet.go

```diff
@@ -117,6 +118,27 @@ func (ksApp *ksApp) Apply(resources kftypes.ResourceEnum) error {
 			return fmt.Errorf("could not change directory to %v Error %v", ksApp.Spec.AppDir, err)
 		}
 	}
+	// Install Istio
```

Comment:
Why move istio to ksonnet part?

Reply:
One benefit is to apply all resources in k8s within ksonnet by default? 
We build k8s config for accessing cluster in ksonnet.

Reply:
I guess it makes sense as we are moving to kustomize. (I don' think we should convert istio manifest to ks)

One thing needs to be done in GCP is create istio namespace.
gcp apply will create secret in namespace `istio-system`, and the namespace is created in `dependencies/istio/install/crds.yaml`

So you need to create that namespace in gcp.apply, and remove the namespace part in  crds.yaml